README.md

```diff
@@ -1626,14 +1626,14 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
         .controller('Avengers', Avengers);
 
     /* @ngInject */
-    function Avengers(storageService, avengerService) {
+    function Avengers(storage, avengerService) {
```

Comment:
Was the suffix removed here because it should be obvious 'storage' is a service even though it is a noun?